tools/workspace-plugin/src/generators/eslint-rule/generator.ts

```diff
@@ -1,11 +1,21 @@
-import { formatFiles, Tree } from '@nx/devkit';
+import { formatFiles, Tree, joinPathFragments, readProjectConfiguration, generateFiles } from '@nx/devkit';
 import { EslintRuleGeneratorSchema } from './schema';
 import { lintWorkspaceRuleGenerator } from '@nx/eslint/src/generators/workspace-rule/workspace-rule';
 
 export async function eslintRuleGenerator(tree: Tree, schema: EslintRuleGeneratorSchema) {
   const options = normalizeSchema(schema);
   await lintWorkspaceRuleGenerator(tree, options);
 
+  const project = readProjectConfiguration(tree, 'eslint-rules');
+  const templateSource = joinPathFragments(__dirname, 'files');
+
+  tree.delete(joinPathFragments(project.root, joinPathFragments(options.directory, `${options.name}.spec.ts`)));
+
+  generateFiles(tree, templateSource, joinPathFragments(project.root, options.directory), {
```

Comment:
this indeed works, but in order to have integration test with nx inner workings we should keep the scaffolding from nx and only update what is necessary within the generated spec file

you either go via AST with tsquery mod https://github.com/microsoft/fluentui/pull/30336/files#diff-11e9f425440da48d0e1dc3b72903c384d0162deaf9aa2651b4cec29e0790caacR26
or simple string replace ( should work for this usecase )